bark/commons/tests/distribution_tests.cc

```diff
@@ -88,15 +90,14 @@ TEST(distribution_test, uniform_dist_1d) {
 
 TEST(distribution_test, multivariate_distribution) {
   // First test zero covariances
-  auto params_ptr = std::make_shared<bark::commons::SetterParams>(true);
-  const double lower_bound = -3.0;
-  const double upper_bound = 10.0;
-  params_ptr->SetListListFloat(
+  auto params_ptr1 = std::make_shared<bark::commons::SetterParams>(true);
+  //const double lower_bound = -3.0;
+  //const double upper_bound = 10.0;
+  params_ptr1->SetListListFloat(
       "Covariance", {{1.0, 0.2, 0.1}, {0.2, 3.0, -0.5}, {0.1, -0.5, 0.125553}});
-  params_ptr->SetListFloat("Mean", {1.2, 12.0, 0.1234f});
-  params_ptr->SetInt("RandomSeed", 1000.0);
-
-  auto dist_multivariate = bark::commons::MultivariateDistribution(params_ptr);
+  params_ptr1->SetListFloat("Mean", {1.2, 12.0, 0.1234f});
+  params_ptr1->SetInt("RandomSeed", 1000.0);
+  auto dist_multivariate = bark::commons::MultivariateDistribution(params_ptr1);
```

Comment:
this distribution never seems to be used? Could omit it in the test

Reply:
Did you expand all code? It is used in line 105 for example.

Reply:
just saw the changes